### PR TITLE
add unique constraints on `name` and `name_inverse`

### PR DESCRIPTION
**Pull request by charbeljc**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

File: partner_relations/model/res_partner_relation_type.py [hunk 1]
```diff
@@ -58,6 +58,13 @@ class ResPartnerRelationType(models.Model):
         'Right partner category',
     )
 
+    _sql_constraints = [
+        ('name_unique', 'unique(name)',
+         'partner relation name already exists!'),
+        ('name_inverse_unique', 'unique(name_inverse)',
+         'partner relation name_inverse already exists!'),
```
Comment (gurneyalex) [inline partner_relations/model/res_partner_relation_type.py]: please remove exclamation marks.

Comment (charbeljc) [inline partner_relations/model/res_partner_relation_type.py]: done !

Comment (charbeljc) [inline partner_relations/model/res_partner_relation_type.py]: ahem ... just added another one ...

**Comment by yvaucher** [comment]
This might require a little update in translations for the error message

**Comment by antespi** [comment]
:+1:

**Comment by charbeljc** [comment]
Hi, I resubmitted this patch again here  #186 , so I close this one. Apologies for the duplicate.

**Pull request closed without merging**